game-service-be/compose.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+  game-service:
+    image: larpex/game-service:latest
+    ports:
+      - "8085:8080"
```

Comment:
` Ports are not available: exposing port TCP 0.0.0.0:8085 -> 0.0.0.0:0: listen tcp 0.0.0.0:8085: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.`

8085 seems to conflict with events service

Reply:
jeez I didn't know events service takes up 3 ports xdddd that's insane

Reply:
fixed